colorcet/plotting.py

```diff
@@ -8,8 +8,6 @@
 
```

Comment:
Probably need a module docstring explaining that this file is entirely optional and only used for the docs, but could be useful for colormaps from any source.

Reply:
Also, shouldn't the sine-comb code be here too?

Reply:
Sure it can be here. I will just give it a name I guess.

Reply:
Eh. It is kind of wierd to import data in there isn't it?

Reply:
Import data?

Reply:
The sine array comes from a numpy file. Do you have the code to generate it?

Reply:
Yes, but unfortunately it is in matlab; sineramp.m in https://www.peterkovesi.com/matlabfns .

Reply:
Looks trivial to translate; up to you!